base/strings/basic.jl

```diff
@@ -403,6 +403,25 @@ julia> lowercase("STRINGS AND THINGS")
 lowercase(s::AbstractString) = map(lowercase, s)
 
 """
+    titlecase(s::AbstractString)
+
+Returns `s` with all initial characters converted to titlecase.
+
+```jldoctest
+julia> titlecase("the julia programming language")
+"The Julia Programming Language"
+```
+"""
+function titlecase(s::AbstractString)
+    words = split(lowercase(s))
```

Comment:
Indeed, the first one sounds more useful, and you can always get the second one by calling `lowercase` first (while the contrary isn't possible).